components/FaqPage/RenderLine.tsx

```diff
@@ -0,0 +1,26 @@
+import DOMPurify from "dompurify";
+
+export const renderLine = (line: string, lineIndex: number) => {
```

Comment:
React component names should begin with a capital letter

```suggestion
export const RenderLine = (line: string, lineIndex: number) => {
```